mesh/mesh.go

```diff
@@ -410,7 +397,7 @@ func (msh *Mesh) ProcessLayer(ctx context.Context, layerID types.LayerID) error
 
 	if !to.Before(from) {
 		if err := msh.pushLayersToState(ctx, from, to, newVerified); err != nil {
-			logger.With().Error("failed to push layers to state", log.Err(err))
+			logger.With().Warning("failed to push layers to state", log.Err(err))
```

Comment:
why warning? this is not expected event in any circumstances

Reply:
it is expected when a certificate is not available. we cannot find a block to apply in that case.

Reply:
changed back to error. it is an error for online nodes.